runelite-client/src/main/java/net/runelite/client/plugins/grounditems/config/ValueThreshold.java

```diff
@@ -0,0 +1,57 @@
+package net.runelite.client.plugins.grounditems.config;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import net.runelite.client.plugins.grounditems.GroundItemsConfig;
+
+@Getter
+@RequiredArgsConstructor
+public enum ValueThreshold
+{
+	LOW("Low", 0),
+	MEDIUM("Medium", 1),
+	HIGH("High", 2),
+	INSANE("Insane", 3),
+	HIGHLIGHTED("Highlighted", 4),
+	OFF("Off", 5);
+
+	private final String name;
+	private final int value;
+
+	@Override
+	public String toString()
+	{
+		return name;
+	}
+
+	public boolean isGreaterThan(ValueThreshold threshold)
+	{
+		return value >= threshold.value;
+	}
+
+	public static ValueThreshold getThreshold(int cost, GroundItemsConfig config)
```

Comment:
I would probably make this work in similar way as colors do atm, so just populate some list and iterate it.